codec-http/src/main/java/io/netty/handler/codec/http/cookie/CookieHeaderNames.java

```diff
@@ -28,6 +28,40 @@
 
     public static final String HTTPONLY = "HTTPOnly";
 
+    public static final String SAMESITE = "SameSite";
+
+    /**
+     * Possible values for the SameSite attribute.
+     * See <a href=https://tools.ietf.org/html/draft-west-cookie-incrementalism-00#section-3.1>changes to RFC6265bis</a>
```

Comment:
Please update to latest RFC6265bis latest draft :)

Reply:
@dvlato ^^